examples/resources/spectrocloud_application_profile/resource.tf

```diff
@@ -1,37 +1,67 @@
-resource "spectrocloud_application_profile" "test_profile_tf"{
-  name = "profile-herbs"
-  version = "1.0.0"
-  context = "project"
-  tags = ["sivaa", "terraform"]
+resource "spectrocloud_application_profile" "app_profile_all_tiers" {
```

Comment:
@SivaanandM We should also add this example in the template file `templates/resources/application_profile.md.tmpl`. That way the documentation will also contain the example.

Reply:
Added example in template file